src/domain/safe/entities/transfer.entity.ts

```diff
@@ -12,11 +11,43 @@ export interface ERC20Transfer extends Transfer {
   value: string;
 }
 
+export function isERC20Transfer(transfer: Transfer): transfer is ERC20Transfer {
+  const erc20Transfer = transfer as ERC20Transfer;
+  return (
+    erc20Transfer.tokenAddress !== undefined &&
+    erc20Transfer.tokenAddress !== null &&
+    erc20Transfer.value !== undefined &&
+    erc20Transfer.value !== null
```

Comment:
Should be equivalent to

```typescript
erc20Transfer.value != null
```

Reply:
[a41f860](https://github.com/5afe/safe-client-gateway-nest/pull/206/commits/a41f8600dc9fbe237de992a74c51dce6a6703668)